src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="big-search-bar"
+      action="/"
+      method="post"
+      data-qa="big"
+    >
+      <div class="search-field-wrapper">
+        <img
+          class="search-icon big-search-icon"
```

Comment:
redundant `big-search-icon` class, you can move all styles under `search-icon` class and modify them for small image in `search-icon--small` class

![image](https://user-images.githubusercontent.com/80700247/232309626-1c728263-411c-4302-a1f2-5b58bf08fcd1.png)

Check all such cases